testutil/daemon/daemon.go

```diff
@@ -621,6 +621,11 @@ func (d *Daemon) Kill() error {
 		return err
 	}
 
+	_, err := d.cmd.Process.Wait()
+	if err != nil && !strings.Contains(err.Error(), "no child processes") {
```

Comment:
```suggestion
	if err != nil && !errors.Is(err, syscall.ECHILD) {
```
FYI, [`ECHILD` is defined on Windows.](https://pkg.go.dev/syscall?GOOS=windows#:~:text=EBFONT%0A%09EBUSY%0A%09ECANCELED-,ECHILD,-ECHRNG%0A%09ECOMM%0A%09ECONNABORTED)